raspbian/applications/tvservice.md

```diff
@@ -0,0 +1,63 @@
+## TVSERVICE
+
+`tvservice` is a command line application used to get and set information about the display, targeted mainly at HDMI video and audio.
+
+Typing `tvservice` by itself will display a list of available command line options.
+
```

Comment:
I think you should explain the major caveat with this command that is destroys any active dispmanx layers.
So, powering off and on the display (or changing resolution) will result in no framebuffer console (and with similar effects on other users of dispmanx layers).

Might be worth pointing to ``vcgencmd display power`` as a less destructive way of disabling display output.